content/guides/end-to-end-testing/auth0-authentication.md

```diff
@@ -169,6 +161,18 @@ Cypress.Commands.add('loginToAuth0', (username: string, password: string) => {
 
   // Ensure Auth0 has redirected us back to the RWA.
   cy.url().should('equal', 'http://localhost:3000/')
+}
+
+Cypress.Commands.add('loginToAuth0', (username: string, password: string) => {
+  const log = Cypress.log({
```

Comment:
do we want to add the logging into the function as well so its encapsulated in the session?